client/package.json

```diff
@@ -145,6 +145,7 @@
 		"fast-json-stable-stringify": "^2.1.0",
 		"filesize": "^6.0.1",
 		"flag-icon-css": "^3.4.5",
+		"framer-motion": "^10.13.0",
```

Comment:
I'm going to check in with the Calypso team about adding this dependency before landing this PR. 

I added it to replicate the behavior of `ToggleGroupControl` in `@wordpress/components`.

Reply:
`motion` from framer-motion is re-exported from `@wordpress/components` as `__unstableMotion`, so it can be used through that. Though, we are [planning](https://github.com/WordPress/gutenberg/issues/60975) to phase it out.

Reply:
Hmm, OK. But if the ambition is for `@wordpress/components` to stop exporting it, would it be better to keep the local dependency..?

Reply:
This doesn't apply anymore